.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Nix
+        uses: DeterminateSystems/nix-installer-action@v4
+
+      - name: Install Devbox
+        uses: jetpack-io/devbox-install-action@v0.4.0
+        with:
+          enable-cache: true
+
+      - name: Check formatting
+        run: devbox run -- composer run-script format-check
```

Comment:
We're only running format step in the main pipeline, since this should be the same across PHP versions.

Reply:
Could be nice to have a `lint` script in `devbox.json` with the goal of only having to run `devbox run lint` in ~any SDK.

Reply:
We do have a `format` script that's consistent across SDKs: https://github.com/workos/workos-php/blob/624eb433b2f502cea1431cb3604abb322e0e35da/devbox.json#L11

But linting and formatting are separate (and the PHP SDK doesn't have a linter setup).

And in CI we want to check the formatting (instead of running the formatter), so we're just invoking the command by hand since it probably doesn't need to be invoked outside of CI.